pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -70,6 +70,8 @@ spec:
                 value: "true"
               - name: port
                 value: "6443"
+              - name: vip_interface
```

Comment:
Ideally we should do unrelated things like this in a separate PR. But then again, this will go away anyway with the new kube-vip version, so it doesn't matter.

Reply:
Agreed! As this is a temp change I thought to include it in this PR. I will keep this in mind from the next time.